local_planner/src/nodes/local_planner.cpp

```diff
@@ -205,10 +206,10 @@ void LocalPlanner::determineStrategy() {
         reach_altitude_ && use_back_off_) {
       if (!back_off_) {
         back_off_point_ = closest_point_;
-        back_off_start_point_ = pose_.pose.position;
+        back_off_start_point_ = toEigen(pose_.pose.position);
```

Comment:
Never been a fan of the `pose_.pose`-business. IMHO `pose_` should be more descriptive. Pose of what? The goal? The drone? Donald Trump? We don't know...

Reply:
I'll keep in mind a more descriptive name for it for the next PR, which will have to change the `pose_` things considerably.